export_test.go

```diff
@@ -75,3 +75,8 @@ func RefImplEvalBody(enc *msgpack.Encoder, expr string, args interface{}) error
 func RefImplExecuteBody(enc *msgpack.Encoder, expr string, args interface{}) error {
 	return fillExecute(enc, expr, args)
 }
+
+// RefImplFutureWait is reference implementation for waiting the Future to be filled.
+func RefImplFutureWait(fut *Future) {
+	fut.wait()
+}
```

Comment:
It's a bad idea to test the internal implementation. There will be bad tests, this will make internal refactoring difficult.

`RefImplExecuteBody`, `RefImpl<Request>Body` is okay here (but not perfect) because we assume that the request body will not change. We can copy-paste a request body creation into test code to test it, but it have no sense.

`SslDialTimout`, `SslCreateContext` is okay here (but not perfect too) because we really don't want to share an integration API with OpenSSL, but should to test the integration.

I don't see why we need to test `fut.wait()` directly.